src/Navigation/Notifications/Factory/Notification.php

```diff
@@ -107,6 +115,12 @@ public function getMessageFromNotification(Entity\Notification $Notification): a
 	{
 		$message = [];
 
+		$cachekey = 'Notification:' . $Notification->id;
```

Comment:
Using a constant instead?

Reply:
Since I'm only calling it from one location , I think a constant is not needed here.